### PR TITLE
JW-1130 Use symlink instead of cp.

### PR DESCRIPTION
**Pull request by m-kostrzewa**

### CODE REVIEW

**Comment by m-kostrzewa** [comment]
When this is merged, move boost subdir from common dir (1 dir up)

**Comment by m-kostrzewa** [comment]
Janie, przetestuj proszę

**Review by mclapinski — CHANGES_REQUESTED** [review]
cmd /c is so 2011

**Review by katrybacka — APPROVED** [review]

File: CIScripts/BuildFunctions.ps1 [hunk 1]
```diff
@@ -39,13 +39,18 @@ function Copy-Repos {
...
+
     Write-Host "Copying common third-party dependencies"
     New-Item -ItemType Directory .\third_party
-    Copy-Item -Recurse "$ThirdPartyCache\common\*" third_party\
+    Get-ChildItem "$ThirdPartyCache\common" -Directory |
+        Where-Object{$_.Name -notlike "boost*"} |
+        Copy-Item -Destination third_party\ -Recurse -Force
+
+    Write-Host "Symlinking boost"
+    New-Item -Path "third_party\boost_1_62_0" -ItemType SymbolicLink -Value "$ThirdPartyCache\boost_1_62_0"
```
Comment (mclapinski) [inline CIScripts/BuildFunctions.ps1:53]: you could change `"$ThirdPartyCache\boost_1_62_0"` to `"$ThirdPartyCache\common\boost_1_62_0"` and delete the folder `$ThirdPartyCache\boost_1_62_0`

**Review by mclapinski — APPROVED** [review]

**Comment by m-kostrzewa** [comment]
Janie, przetestuj proszę

File: CIScripts/BuildFunctions.ps1 (hunk 1 above)
Comment (m-kostrzewa) [reply CIScripts/BuildFunctions.ps1:53]: Will leave as it is for now so it'll be easier to revert if it breaks something

JW-1140 - Todo, in next sprint

**Pull request merged**